Common/src/main/java/at/petrak/hexcasting/common/lib/HexCreativeTabs.java

```diff
@@ -20,11 +21,14 @@ public static void registerCreativeTabs(BiConsumer<CreativeModeTab, ResourceLoca
 
     private static final Map<ResourceLocation, CreativeModeTab> TABS = new LinkedHashMap<>();
 
-    public static final CreativeModeTab HEX = register("hexcasting", CreativeModeTab.builder(CreativeModeTab.Row.TOP, 7)
+    public static final CreativeModeTab MAIN = register("main", CreativeModeTab.builder(CreativeModeTab.Row.TOP, 7)
```

Comment:
we should keep this variable the same probably to avoid breaking any addons that wanna dump their junk into hex's inventory (i know atleast hexal does, i think others might?)